iam_user.tf

```diff
@@ -0,0 +1,43 @@
+resource "aws_iam_user" "default" {
```

Comment:
Can we use this module instead: https://github.com/cloudposse/terraform-aws-iam-system-user

Reply:
@osterman,  in my case user will be create if `username` is defined,  with module `terraform-aws-iam-system-user` can't get same functional. The one way is add `count` to module `terraform-aws-iam-system-user`.

Reply:
@osterman , I made PR in https://github.com/cloudposse/terraform-aws-iam-system-user/pull/2#issuecomment-342165056